### PR TITLE
Fix app types

### PR DESCRIPTION
**Pull request by klzns**
#### What problem is this solving?

When importing the components from other apps the types were not working.

#### How to test it?

https://pricets--storecomponents.myvtex.com/
https://brenovtex--carrefourbrfood.myvtex.com/busca/panetone
https://brenovtex--motorolauk.myvtex.com/
https://brenovtex--dzarm.myvtex.com/novidades
https://pricets--sandboxusdev.myvtex.com/

#### Screenshots or example usage:

Installments List is also working, that is the piece of code that had the most refactor:
https://pricets--storecomponents.myvtex.com/vintage-phone/p

image

### CODE REVIEW

File: react/components/InstallmentsContext.ts [hunk 1]
```diff
@@ -1,15 +1,8 @@
 import { createContext, useContext } from 'react'
-
-export interface Installments {
```
Comment (klzns) [inline react/components/InstallmentsContext.ts:3]: This type as written as plural (installments) but it refer to a singular installment.

**Review by icazevedo — APPROVED** [review]
LGTM, although the Installments List is showing everything in the same line.

Screen Shot 2020-12-21 at 2 54 26 PM

**Review by kaisermann — APPROVED** [review]

File: react/ListPrice.tsx [hunk 2]
```diff
@@ -15,8 +14,24 @@ const CSS_HANDLES = [
...
+  title: {
+    id: 'admin/list-price.title',
+  },
+  description: {
+    id: 'admin/list-price.description',
+  },
+  default: {
+    id: 'store/list-price.default',
+  },
+})
```
[the remark this answers is not in the document]
Comment (klzns) [reply react/ListPrice.tsx:32]: I can't, I use the variable `messages` in default props

**Review by kaisermann — APPROVED** [review]

File: react/Installments.tsx [hunk 3]
```diff
@@ -22,42 +37,30 @@ const Installments: StorefrontFC<BasicPriceProps> = props => {
     return null
   }
 
-  let maxInstallments: ProductTypes.Installment | undefined
+  let [maxInstallment] = commercialOffer.Installments
```
Comment (victorhmp) [inline react/Installments.tsx:40]: I think changing this from `maxInstallments` to `maxInstallment` could be misleading. The max _installments_, would be the payment options with the highest number of individual installments. While the max _installment_ would be the highest individual installment value, which in this case is the opposite of what you want, right?

File: react/InstallmentsList.tsx [hunk 4]
```diff
@@ -40,9 +32,9 @@ function InstallmentsList(props: Props) {
 
   return (
     <div className={handles.installmentsListContainer}>
-      {pickedInstallments.map((inst: Installments, i: number) => {
+      {pickedInstallments.map((inst, i: number) => {
         return (
-          <InstallmentsItem installments={inst} key={i}>
+          <InstallmentsItem installment={inst} key={i}>
```
Comment (victorhmp) [inline react/InstallmentsList.tsx:37]: Also think changing this from `installments` to `installment` is not a good idea

File: react/InstallmentsList.tsx [hunk 5]
```diff
@@ -51,14 +43,13 @@ function InstallmentsList(props: Props) {
   )
 }
 
-interface InstallmentsItemProps {
-  children: React.ReactNode
-  installments: Installments
+interface Props {
+  installment: ProductTypes.Installment
```
Comment (victorhmp) [inline react/InstallmentsList.tsx:47]: Same thing

**Review by victorhmp — CHANGES_REQUESTED** [review]

File: react/InstallmentsList.tsx [hunk 6]
```diff
@@ -40,9 +32,9 @@ function InstallmentsList(props: Props) {
 
   return (
     <div className={handles.installmentsListContainer}>
-      {pickedInstallments.map((inst: Installments, i: number) => {
+      {pickedInstallments.map((inst, i: number) => {
```
Comment (Klynger) [inline react/InstallmentsList.tsx:35]:
```suggestion
      {pickedInstallments.map((inst, i) => {
```

**Review by Klynger — APPROVED** [review]

**Review by victorhmp — APPROVED** [review]

**Pull request merged**